src/main/java/manufacturing/service/impl/CarServiceImpl.java

```diff
@@ -0,0 +1,65 @@
+package manufacturing.service.impl;
+
+import java.util.List;
+import manufacturing.dao.CarDao;
+import manufacturing.dao.DriverDao;
+import manufacturing.injections.Inject;
+import manufacturing.injections.Service;
+import manufacturing.model.Car;
+import manufacturing.model.Driver;
+import manufacturing.service.CarService;
+
+@Service
+public class CarServiceImpl implements CarService {
+
+    @Inject
+    private CarDao carDao;
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id).orElseThrow(() ->
+                new RuntimeException("Car with id " + id + " not found"));
+    }
+
+    @Override
+    public List<Car> getAll() {
+        return carDao.getAll();
+    }
+
+    @Override
+    public Car update(Car car) {
+        return carDao.update(car);
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return carDao.delete(id);
+    }
+
+    @Override
+    public void addDriverToCar(Driver driver, Car car) {
+        car.getDrivers().add(driver);
+        carDao.update(car);
+        if (!driverDao.getAll().contains(driver)) {
```

Comment:
when we connect a db to our project it will handle this cases and throw exception if driver is not present. Let's remove this check